packages/ses/src/sloppy-globals-scope-terminator.js

```diff
@@ -18,7 +18,7 @@ import {
  * to prevent access to "start compartment globals". When "sloppyGlobalsMode"
  * is true, the Proxy will perform sets on the "globalObject".
  */
-export const createSloppyGlobalsScopeTerminator = globalObject => {
+export const createSloppyGlobalsScopeTerminator = freeze(globalObject => {
```

Comment:
This may interfere with certain debuggers’ ability to infer the name of the function. That can be addressed by freezing on the subsequent line.